src/styles/components/_map.scss

```diff
@@ -3,23 +3,28 @@
 .card-map {
   outline: $outline-border;
   border-radius: $border-radius-lg;
-
+ 
   .card-body {
     padding: 1.5rem;
-
+ 
     .card-title {
       @extend .headline-lg;
       margin-bottom: 0;
     }
-
+ 
     .card-text {
       margin-top: 0.25rem;
     }
-
+ 
     .card-iframe {
       max-height: 146px;
       margin: 1rem 0;
-
+ 
+      @media screen and (width >=768px) and (width <=1200px) {
```

Comment:
se podria usar 
Between breakpoints 
Similarly, media queries may span multiple breakpoint widths:


@include media-breakpoint-between(md, xl) { ... }
Which results in:


// Example
// Apply styles starting from medium devices and up to extra large devices
@media (min-width: 768px) and (max-width: 1199.98px) { ... }

Reply:
Ready✅

Reply:
aca no esta cambiado